spec/controllers/hiring_staff/vacancies/application_controller_spec.rb

```diff
@@ -71,7 +71,7 @@
 
       it 'converts date params to a Date object' do
         subject.convert_multiparameter_attributes_to_dates(:test_form, [:starts_on])
-        expect(controller.params[:test_form][:starts_on]).to eql(Date.new(2020, 01, 01))
+        expect(controller.params[:test_form][:starts_on]).to eql(Date.new(2020, 0o1, 0o1))
```

Comment:
What does `0o1` mean?

Reply:
I run rubocop with autocorrect and this is one of the things that was fixed with that. apparently it thinks that those are octal number

```
spec/controllers/hiring_staff/vacancies/application_controller_spec.rb:74:85: C: Style/NumericLiteralPrefix: Use 0o for octal literals. (https://rubystyle.guide#numeric-literal-prefixes)
        expect(controller.params[:test_form][:starts_on]).to eql(Date.new(2020, 01, 01))
```